code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -287,12 +285,12 @@
 	heat_level_2 = 480 //Default 400
 	heat_level_3 = 1100 //Default 1000
 
-	//reagent_tag = IS_SKRELL //VOREStation Edit
-
 	breath_heat_level_1 = 400	//Default 380 - Higher is better
 	breath_heat_level_2 = 500	//Default 450
 	breath_heat_level_3 = 1350	//Default 1250
 
+	reagent_tag = IS_SKRELL
```

Comment:
Unless you understand what this does, change this back, because I'm pretty sure all this does is make Skrell sick for eating meats, which is aggravating as hell.